sqtom/tests/test_fitting_1d.py

```diff
@@ -16,7 +16,53 @@
 import pytest
 import numpy as np
 from sqtom.forward_solver import degenerate_pmf
-from sqtom.fitting_1d import fit_1d, threshold_1d, marginal_calcs_1d
+from sqtom.fitting_1d import (
+    two_schmidt_mode_guess,
+    marginal_calcs_1d,
+    gen_hist_1d,
+    threshold_1d,
+    fit_1d,
+)
+
+
+@pytest.mark.parametrize("sq_0", [0.1, 0.5, 1.0])
+def test_gen_hist_1d(sq_0):
```

Comment:
@DavidSPhillips : I have significantly simplified this test only checking for lossless distributions, this reduces the time it takes to run the tests significantly without reducing the coverage.